decktape.js

```diff
@@ -365,6 +365,7 @@ async function exportSlide(plugin, page, printer, context) {
     printBackground     : true,
     pageRanges          : '1',
     displayHeaderFooter : false,
+    scale: 1.5
```

Comment:
I think it's related to #107 and specific to Impress. I would suggest we add an option and change the default value for Impress instead of hard-coding the value for all presentation frameworks.